command/format.go

```diff
@@ -408,3 +425,19 @@ func looksLikeDuration(k string) bool {
 		k == "duration" || strings.HasSuffix(k, "_duration") ||
 		k == "lease_max" || k == "ttl_max"
 }
+
+// This struct is responsible for capturing all the fields to be output by a
+// vault status command, including fields that do not come from the status API.
+// Currently we are adding the fields from api.LeaderResponse
+type SealStatusOutput struct {
+	api.SealStatusResponse
+	HAEnabled                bool   `json:"ha_enabled"`
```

Comment:
Any reason you didn't embed http.LeaderResponse in the same way as SealStatusResponse?

Reply:
Yeah, so I wanted the fields in http.LeaderResponse to all have `omitempty` set, except for the HAEnabled field, so that empty values would not be marshaled into the format. Because of that, I copied those fields over and added `omitempty` to them individually.

Reply:
Got it, thanks.